### ISSUE TITLE
Prometheus Scrape Error

### ISSUE THREAD

**Issue by mkl262** [google/cadvisor#2133]
Hello,

I Have encountered an issue with Prometheus and the latest [canary tag docker](https://hub.docker.com/r/google/cadvisor) where it fails to scrape the target with the following error: 

> expected timestamp or new record, got "MNAME"

![screenshot_20181223_214509](https://user-images.githubusercontent.com/8472607/50387027-16780380-06fc-11e9-8383-441564aac7a3.png)

Prometheus Configuration: 

> - job_name: cAdvisor
>   scrape_interval: 5s
>   static_configs:
>   - targets: ["wb-cadvisor:8080"]

**cAdvisor version:**  v0.32.0.17+864d279257bb4b

**cAdvisor commit:**  [864d279](https://github.com/google/cadvisor/commit/864d279257bb4be26b4433af89c6f3272b1dbd76)

**Comment by Betula-L** [comment]
Execuse me, this bug may be introduced by my PR https://github.com/google/cadvisor/pull/2124. I'm trying to fix it.

**Comment by Betula-L** [comment]
This bug is introduced by my PR https://github.com/google/cadvisor/pull/2124. I'm trying to fix it as well as add more testcase.

**Comment by Betula-L** [comment]
**Why this issue?**

@dashpole  PR https://github.com/google/cadvisor/pull/2124 led an error behavior when prometheus scrapes /metrics. 

It's because i ignored metrics `container_last_seen` at prometheus.go:138, and `ignoreRe` at prometheus_test.go:271 ignores it also. Thus, the bad effect of https://github.com/google/cadvisor/pull/2124 was not detected. 

Current metric `container_last_seen` is as follow,
```
container_last_seen{id="/system.slice/avahi-daemon.service",image="",name=""} 1.546053157e+09 -62135596800000
```
Revised metrics `container_last_seen` of https://github.com/google/cadvisor/pull/2135
```
container_last_seen{id="/system.slice/avahi-daemon.service",image="",name=""} 1.546053157e+09 1546053157000
```
**Furthermore**
1. I don't think `container_last_seen` should be metric at namespace `container`. It could be individual metrics out of func `collectContainersInfo`. 
2. prometheus_test.go ought to be improved.

Hope for insightful solution.

**Comment by CptDayDreamer** [comment]
Is it fixed? Because I still got the bug now with Prometheus...

**Comment by dashpole** [comment]
I believe this was fixed by #2135.  Feel free to open a new issue.
